### PR TITLE
Maker Order Placement

### LINKED ISSUE #180: [Sim Exchange] Create functionality to allow users to place orders (as maker)
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

### User Story

As a sim exchange user I would like to be able to place orders (act as a maker, providing liquidity) 

### Why Is this Needed?
*Summary*: For a more fully featured beta

### Description
*Type*: Feature

### Definition of Done

- [x] Create a basic UI that allows for users to select a contract, qty, price, and expiration to create an order
- [ ] Create needed call to sign the order via web3
- [ ] All functionality should be done using MARKET.js
- [ ] Eventually this newly created and signed order will be transmitted to our Sim Exchange orderbook to be filled (no action needed for this issue)

### Additional Information
We can refine the UI later, this ticket is mainly for the underlying functionality.

### PR DESCRIPTION
**Pull request by travisdmathis**
##### Description

This PR introduces the ability for users to create maker orders from within the dapp.  It also refactors how the dapp uses MARKET.js for less code repetition.

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)

UI, UX, Tests

##### Refers/Fixes

Fixes #180

### CODE REVIEW

**Comment by eswarasai** [comment]
@travisdmathis @pelsasser -- Please hold off on merging this branch until we can get the other PR #198 wrapped up. There were already couple of conflicts. If we merge this PR, I'll be clueless on how to resolve them. Thanks!

**Comment by travisdmathis** [comment]
@eswarasai no problem, I still have some work to do so you got some time :)

**Comment by eswarasai** [comment]
@travisdmathis -- Cool. I've just merged the other PR. So you might have to rebase your branch from `develop`. I'll suggest holding off on rebase until you're dev done on this and ready to be merged. Thanks!

**Comment by travisdmathis** [comment]
@eswarasai thanks for the heads up..

File: src/util/utils.js [hunk 1]
```diff
@@ -341,6 +174,37 @@ export const isTestnetOrMainnet = network => {
...
 
+/**
+ * Get an ERC20 Token Balance
+ *
+ * @param tokenAddress
+ * @param toString
+ * @param callback function for returning response from the web3 callback
+ * @return BigNumber or String balance of users metamask address
+ */
+export const getTokenBalance = (tokenAddress, toString, callback) => {
```
Comment (pelsasser) [inline src/util/utils.js]: once we get the new MARKET.js integrated, we can use the functionality in there for some of the ERC20 interactions.

**Comment by pelsasser** [comment]
@travisdmathis - feel free to merge if we can figure out all the conflicts.  Sorry.

**Pull request merged**